style.css

```diff
@@ -499,6 +504,114 @@ fieldset {
   align-items: center;
 }
 
+.modal {
+  display: none;
+}
+
+.modal.active {
+  position: fixed;
+  height: 90vh;
+  width: 100%;
+  overflow-y: auto;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  z-index: 5;
+}
+
+.popupWindow {
+  align-self: center;
+  background-color: rgba(255, 255, 255, 0.16);
```

Comment:
- [Optional] Your modal background is transparent making it difficult to read text, this is not User friendly, Kindly take note of this